src/plugins/discover/public/application/components/discover_grid/constants.ts

```diff
@@ -9,7 +9,6 @@
 // data types
 export const kibanaJSON = 'kibana-json';
 export const geoPoint = 'geo-point';
-export const unknownType = 'unknown';
```

Comment:
we treat this type in the same way with `kibanaJSON` type. No need to duplicate code